lib/ansible/plugins/action/copy.py

```diff
@@ -29,7 +29,6 @@
 from ansible.utils.boolean import boolean
 from ansible.utils.hashing import checksum
 from ansible.utils.unicode import to_bytes
-from ansible.parsing.vault import VaultLib
```

Comment:
it's a simple thing to re-add when we have some code that actually makes use of it (and if we implement this via VaultFile, it won't even be the same line that we need).